<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphShowWindow</class>
 <widget class="QWidget" name="GraphShowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,12,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,2,2,2,2,2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_theme">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
						border:2px solid;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(0, 0, 0);
border-color:rgb(0, 0, 0);
						 }

						 QPushButton:hover {
							 color: rgb(52, 167, 255);
							 border-color: rgb(52, 167, 255);
						 }

						 QPushButton:pressed {
							 color: rgb(46, 39, 239);
							 border-color: rgb(46, 39, 239);
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
           </property>
           <property name="text">
            <string>主题</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backgroundSet">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
						border:2px solid;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(0, 0, 0);
border-color:rgb(0, 0, 0);
						 }

						 QPushButton:hover {
							 color: rgb(52, 167, 255);
							 border-color: rgb(52, 167, 255);
						 }

						 QPushButton:pressed {
							 color: rgb(46, 39, 239);
							 border-color: rgb(46, 39, 239);
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
           </property>
           <property name="text">
            <string>设置背景图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_shadow">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
						border:2px solid;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(0, 0, 0);
border-color:rgb(0, 0, 0);
						 }

						 QPushButton:hover {
							 color: rgb(52, 167, 255);
							 border-color: rgb(52, 167, 255);
						 }

						 QPushButton:pressed {
							 color: rgb(46, 39, 239);
							 border-color: rgb(46, 39, 239);
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
           </property>
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_top">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
						border:2px solid;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(0, 0, 0);
border-color:rgb(0, 0, 0);
						 }

						 QPushButton:hover {
							 color: rgb(52, 167, 255);
							 border-color: rgb(52, 167, 255);
						 }

						 QPushButton:pressed {
							 color: rgb(46, 39, 239);
							 border-color: rgb(46, 39, 239);
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
           </property>
           <property name="text">
            <string>置于顶层</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportPushbuttom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
						border:2px solid;
border-radius:5px;
background-color: rgba(221, 221, 221, 0);
color: rgb(0, 0, 0);
border-color:rgb(0, 0, 0);
						 }

						 QPushButton:hover {
							 color: rgb(52, 167, 255);
							 border-color: rgb(52, 167, 255);
						 }

						 QPushButton:pressed {
							 color: rgb(46, 39, 239);
							 border-color: rgb(46, 39, 239);
						 }

						 QPushButton:disabled {
							 color: #333333;
						 }</string>
           </property>
           <property name="text">
            <string>导出为</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GraphicsView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="resizeAnchor">
          <enum>QGraphicsView::AnchorViewCenter</enum>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::FullViewportUpdate</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <widget class="QLabel" name="cursorCoor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphShow/GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
